src/seedu/addressbook/commands/ListUsersCommand.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class ListUsersCommand extends Command{
+
+    public static final String COMMAND_WORD = "list_users";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all created users and password.\n"
```

Comment:
Is it secure to list all password xD

Reply:
@Happytreat is right. Perhaps you can list all users only?